backend/src/resolvers/comments.js

```diff
@@ -0,0 +1,12 @@
+import { neo4jgraphql } from 'neo4j-graphql-js'
+
+export default {
+  Mutation: {
+    CreateComment: (object, params, context, resolveInfo) => {
+      return neo4jgraphql(object, params, context, resolveInfo, false)
+    },
+    AddPostComments: (object, params, context, resolveInfo) => {
```

Comment:
Please do not create a Comment without a Post. The frontend should assign the Post of the Comment already during creation. We don't want any left-over state, ie. comments without authors or posts.
```suggestion
```

Reply:
I do not want to leave it like this @roschaefer, which is why I said :point_up: I am not happy with it, but I spent a couple hours last night and could not seem to get it working, and there are no examples of creating any nodes passing in additional information.

We have a few where we assign an author, but that does not require passing in any additional info as we can access the user id with the context. 

When I try to modify the resolver, I run into the issues stated :point_up: